tests/integration/test_backup_restore_on_cluster/test_disallow_concurrency.py

```diff
@@ -125,19 +125,29 @@ def test_concurrent_backups_on_same_node():
         .query(f"BACKUP TABLE tbl ON CLUSTER 'cluster' TO {backup_name} ASYNC")
         .split("\t")[0]
     )
-    assert_eq_with_retry(
-        nodes[0],
-        f"SELECT status FROM system.backups WHERE status == 'CREATING_BACKUP' AND id = '{id}'",
-        "CREATING_BACKUP",
```

Comment:
Here was the first problem - sometimes a backup is created so fast that the initial status in this line is `BACKUP_CREATED`, and not `CREATING_BACKUP`.